src/kajal/ScannerExamples/Assignment14p2.java

```diff
@@ -0,0 +1,57 @@
+package kajal.ScannerExamples;
+
+import java.util.Scanner;
+
+public class Assignment14p2 {
+
+		void Vowels(char alpha) {
+		
```

Comment:
FOr this program take reference of below code

Aparana - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/533/files